### PR TITLE
Page Register Penerima Dana, Register Penerima Datadiri dan Controller Register Penerima Dana , Register Penerima DatadiriRegister penerima datadiri dan penerima dana page

### PR DESCRIPTION
**Pull request by irfan2543**
Page Register Penerima Dana, Register Penerima Datadiri dan Controller Register Penerima Dana , Register Penerima Datadiri

### CODE REVIEW

**Comment by alpinmohsa541** [comment]
fields terlalu besar ka
image

**Comment by alpinmohsa541** [comment]
**Masih eror**

**Comment by alpinmohsa541** [comment]
Erorr role=lender
image
role=borrower
image

**Comment by alpinmohsa541** [comment]
Lender masih eror
image

File: routes/web.php [hunk 1]
```diff
@@ -19,18 +22,14 @@
...
-        return view('dashboard');
-    })->name('dashboard');
+        return view('dashboard');})->name('dashboard');
 });
 
 Route::get('/register-role', [RegisterRoleController::class, 'registerRole'])->name('register-role');
 Route::get('/register-borrower', [RegisterRoleController::class, 'RegisterBorrowerPage'])->name('register-borrower');
-
+Route::get('/register-penerima-datadiri', [RegisterPenerimaDatadiriController::class, 'registerPenerimaDatadiri'])->name('register-penerima-datadiri');
+Route::get('/register-penerima-dana', [RegisterPenerimaDanaController::class, 'registerpenerimadana'])->name('register-penerima-dana');
```
Comment (zippo538) [inline routes/web.php:34]: controller nya blm ada mas

**Review by zippo538 — APPROVED** [review]
succcess

**Pull request merged**